src/redux/sagas.ts

```diff
@@ -13,13 +13,9 @@ function* fetchData(action: ReturnType<typeof changeInput>) {
     )
     if (!cachedData) {
       const response: AxiosResponse<SickResponseData> = yield sick.getSick(action.payload)
+      console.log('api called!')
```

Comment:
```suggestion
      console.info('api called!')
```

<img width="724" alt="image" src="https://github.com/pre-onboarding-12th-1team/pre-onboarding-12th-1-3/assets/74622889/220ea19a-cc89-4955-ae6f-659136785b6f">

요구사항은 console.info라서 수정이 필요합니다!